airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/command/InputStateFactory.kt

```diff
@@ -62,7 +61,7 @@ class InputStateFactory {
                     }
                     groupValues.last()
                 }
-        val nonGlobalStreams: Map<AirbyteStreamNameNamespacePair, OpaqueStateValue> =
```

Comment:
We have something really close to this, I'll look into whether we can/should share it.

Reply:
Agreed, that's worth investigating at least.